### PR TITLE
4.0.x overwrite repo names

### PR DESCRIPTION
**Pull request by Jey-Cee**
Changes:
default -> Stable (default)
latest -> Beta (latest)

### CODE REVIEW

**Comment by Jey-Cee** [comment]
#623

File: src/js/adminSystem.js [hunk 1]
```diff
@@ -126,9 +126,14 @@ function System(main) {
             for (var repo in that.systemRepos.native.repositories) {
                 if (!that.systemRepos.native.repositories.hasOwnProperty(repo)) continue;
                 var obj = that.systemRepos.native.repositories[repo];
-
+                let name = repo;
+                if (repo === 'default') {
+                    name = 'Stable (default)';
+                } else if (repo === 'latest') {
+                    name = 'Beta (latest)';
+                }
```
Comment (AlCalzone) [inline src/js/adminSystem.js:134]: I think this comparison should be extracted to a function, since you're using it three times.

**Comment by AlCalzone** [comment]
Do we need to port this to the master branch aswell?

**Comment by Apollon77** [comment]
> Do we need to port this to the master branch aswell?
Yes, AND we need to create a secind issue (or move the initila one) so that it is ALSO implemented in the react code

**Comment by GermanBluefox** [comment]
Thank you!

**Pull request merged**